### DISCUSSION TITLE
PyCon India 2023: Logo Typography

### DISCUSSION THREAD

**Discussion by inovizz** [pythonindia/inpycon2023-tasks#33] [General]
Folks, the Webchirpy team has sent the typography for the logo of PyCon India 2023. Please review the items, and vote on the options.

Deadline to put your votes: May 16th, 2023. 5:00 pm IST.

Typography 👇🏽👇🏽👇🏽

Credits: - https://webchirpy.com/

**Comment by inovizz** [comment]
**Option - 1**

Logo
![image](https://github.com/pythonindia/inpycon2023-tasks/assets/3849885/1e50612a-a21c-4926-86cf-73afd4bb435e)

Typography

![image](https://github.com/pythonindia/inpycon2023-tasks/assets/3849885/19d2d3cc-962a-4de2-b394-9927cfa2a4a2)

Print version

![image](https://github.com/pythonindia/inpycon2023-tasks/assets/3849885/7e714d28-d260-4418-babb-c8974c2de423)

**Comment by inovizz** [comment]
**Option - 2**

Logo

![image](https://github.com/pythonindia/inpycon2023-tasks/assets/3849885/b577125d-5403-45e0-b870-bee6c8589dab)

Typography

![image](https://github.com/pythonindia/inpycon2023-tasks/assets/3849885/b1c2c2e4-1c95-4def-9c55-33a5a397256c)

Print Version

![image](https://github.com/pythonindia/inpycon2023-tasks/assets/3849885/4fbd4dc0-7466-469f-8a05-a9ec4d93f9f7)

**Comment by Beingfarazkhan** [comment]
Hello everyone I would like to propose an idea, how about we use colors from our national flag?
One Orange-Green combination would also be a good choice to add to the logo and typography.

**Reply by Animesh-Ghosh** [reply]
Might be cool if we pull it off similar to [PyCon Uganda](https://ug.pycon.org/). If not, I am leaning towards Option 1.

**Comment by Pavan-Nagulla** [comment]
Hello everyone , i would love to have a  colour of black and blue for pycon !!

**Comment by inovizz** [comment]
Thanks, everyone for your input & votes.

Unfortunately, at this moment we are not in a position to accommodate suggestions w.r.t. colour or design changes.

Based on the number of votes, option 1 seems a clear winner. So we'll go with it as PyCon India 2023 logo.
